bindings/java/java/src/test/java/org/ethereum/evmc/EvmcTest.java

```diff
@@ -3,14 +3,28 @@
 // Licensed under the Apache License, Version 2.0.
 package org.ethereum.evmc;
 
+import java.net.URL;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 final class EvmcTest {
-  private static final String exampleVmPath =
-      System.getProperty("user.dir") + "/../c/build/lib/libexample-vm.so";
+  private static final String extension;
+  private static final String exampleVmPath;
+  static {
+    String os = System.getProperty("os.name", "generic").toLowerCase();
+    if (os.contains("mac") || os.contains("darwin")) {
```

Comment:
This seems inconsistent with the other helper that figures out the DLL extension.

Reply:
Good catch, fixed.